decidim-meetings/lib/decidim/meetings/feature.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+Decidim.register_feature(:meetings) do |feature|
+  feature.engine = Decidim::Meetings::ListEngine
+  feature.admin_engine = Decidim::Meetings::AdminEngine
+
+  feature.on(:create) do |instance|
+    Decidim::Meetings::CreateMeeting.call(instance) do
```

Comment:
This doesn't make sense, at `decidim-pages` it was OK since each feature could only have one page.